src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskAssigneeEntity.kt

```diff
@@ -0,0 +1,19 @@
+package fi.metatavu.lipsanen.projects.milestones.tasks
+
+import jakarta.persistence.*
+import java.util.*
+
+@Entity
+@Table(name = "task_assignee")
+class TaskAssigneeEntity {
+
+    @Id
+    lateinit var id: UUID
+
+    @ManyToOne
+    @JoinColumn(name = "task_id", nullable = false)
```

Comment:
```suggestion
    @JoinColumn
```
It's not really nullable, is it? Since we are using lateinit we imply it's not going to be null. "task_id" is not necessary